packages/cmr-client/README.md

```diff
@@ -0,0 +1,253 @@
+# @cumulus/cmr-client
```

Comment:
This just gets rebuilt ?  same problems above in here.

Reply:
Yeah, this is dynamically generated.  @kkelly51 will just need to run `npm run build-docs` to regenerate it once the  JSDocs have been fixed.

Reply:
updated